indico/htdocs/js/indico/modules/contributions/common.js

```diff
@@ -193,6 +196,107 @@
         });
     }
 
+    function onSuccessCallback($element, api) {
+        return function (data) {
+
+            function updateContent(data) {
+                if (data) {
+                    api.set('content.text', ajaxifyForm($(data.html)));
+                    if (data.js) {
+                        $('body').append(data.js);
```

Comment:
But it is not `ajaxqbubble` it is `qbubble` and it wasn't designed to be used only with forms and ajax requests.